rholang/src/main/scala/coop/rchain/rholang/interpreter/match.scala

```diff
@@ -648,11 +648,21 @@ object SpatialMatcher {
             _ <- spatialMatch(t1, p1)
             _ <- spatialMatch(t2, p2)
           } yield Unit
+        case (EPercentPercentBody(EPercentPercent(t1, t2)), EPercentPercentBody(EPercentPercent(p1, p2))) =>
+          for {
+            _ <- spatialMatch(t1, p1)
+            _ <- spatialMatch(t2, p2)
+          } yield Unit
         case (EPlusBody(EPlus(t1, t2)), EPlusBody(EPlus(p1, p2))) =>
           for {
             _ <- spatialMatch(t1, p1)
             _ <- spatialMatch(t2, p2)
           } yield Unit
+        case (EPlusPlusBody(EPlusPlus(t1, t2)), EPlusPlusBody(EPlusPlus(p1, p2))) =>
+          for {
+            _ <- spatialMatch(t1, p1)
+            _ <- spatialMatch(t2, p2)
+          } yield Unit
```

Comment:
@guardbotmk3 I don't know why my comment isn't displayed directly under what I commented on. I was talking about these changes.

Reply:
So, these can match, because they may match inside a body. They just won't ever match at the top level.